consensus/polybft/stake_manager_test.go

```diff
@@ -98,7 +91,7 @@ func TestStakeManager_PostBlock(t *testing.T) {
 					stakeManager.validatorSetContract,
 					validators.getValidator(initialSetAliases[firstValidator]).Address(),
 					types.ZeroAddress,
-					zeroStake,
+					1, // remove initial 1
```

Comment:
This comment is a bit confusing? Should it be like this?
```suggestion
					1, // stake 1 initially
```

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1461/commits/758c3961fb1ae40fb85d7e8a871386ef4479658c